### PR TITLE
Added course model

### PR DESCRIPTION
**Pull request by LiorRey**
Relating to #35 

Added a SQLAlchemy table/class for handling the data of a course.

### CODE REVIEW

File: tutor/models/course.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from .. import db
+
+class Course(db.Model):
+    __tablename__ = 'course'
+    id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String(100), nullable=False)
+    resources = db.relationship('Resource', backref='course', lazy=True)
```
Comment (mosesnagar) [inline tutor/models/course.py:7]: That relationship isn't related yet so it doesn't need to appear here now.

**Comment by bardielle** [comment]
@LiorRey  what about that PR? Is it still relevant?

**Comment by LiorRey** [comment]
> @LiorRey what about that PR? Is it still relevant?

Finished it, waiting for a review.

**Comment by Michal-Shamir** [comment]
@LiorRey  the course and resource model were already merged in the project , 
So isn't it another PR doing the same? maybe you should open new one that works on the existing master branch and adds more functionalities to those models?

**Pull request closed without merging**